js/index.js

```diff
@@ -80,15 +131,21 @@ fetch('words.json')
         const characterSet = 'abcdefghijklmnopqrstuvwxyz'
 
         document.addEventListener('keyup', function (event) {
-            if (characterSet.includes(event.key.toLowerCase()) && gameState.attemptedWord.length < 5 ) {
-                gameState.attemptedWord.push(event.key.toUpperCase())
-                document.querySelector('.tile' + gameState.characterCounter).textContent = event.key
-                gameState.characterCounter++
-            } else if (event.key == 'Backspace') {
-                deletePressed(gameState)
-            } else if (event.key == 'Enter') {
-                enterPressed(gameState)
+            if (!gameState.success) {
+                if (characterSet.includes(event.key.toLowerCase()) && gameState.attemptedWord.length < 5 ) {
+                    gameState.attemptedWord.push(event.key.toUpperCase())
+                    document.querySelector(`.row${gameState.attemptCounter} .tile${gameState.characterCounter}`).textContent = event.key
```

Comment:
It seems like this selector is repeated quite a few times in the code:

`.row${gameState.attemptCounter} .tile${gameState.characterCounter}`

Would be worth switching this over to a function that takes in the gameState and returns the query string

Reply:
done